src/python/qeopenfermion/_utils.py

```diff
@@ -19,6 +19,7 @@
 from zquantum.core.measurement import ExpectationValues, expectation_values_to_real
 from openfermion import count_qubits
 import itertools
+from cirq import LineQubit, GridQubit, PauliSum, PauliString, Z, Y, X
```

Comment:
So far we've been using `PauliSum`s from pyQuil, so this might be confusing to see different Pauli Sums in different places in codebase.
I have two ideas on how to deal with this ambiguity:
a) Use `cirq.PauliSum` and `pyquil.PauliSum` instead.
b) Let it be ;)

Also, I don't really think this is a huge issue at the moment, WDYT?

Reply:
Oh whoops. Didn't even notice pyquil's version is also `PauliSum`. Let's go with option a)

Reply:
Turns out the pyquil imports aren't even used. Removed them and addressed comments from above in: `a307712`